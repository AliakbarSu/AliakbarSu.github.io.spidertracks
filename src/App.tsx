import './App.css'
import Customers from './views/Customers'

function App() {

  return (
    <>
      <div className="border-b border-gray-200 pb-5">
        <h3 className="text-base font-semibold leading-6 text-gray-900">Customers</h3>
      </div>
      <Customers />
    </>
  )
}

export default App
