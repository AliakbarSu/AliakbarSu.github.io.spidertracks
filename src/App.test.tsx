import { describe, it, expect } from 'vitest';
import App from "./App"

describe('App Component', () => {
    it('Should render correctly', () => {
        const component = <App />
        expect(component).toMatchSnapshot();
    })
});