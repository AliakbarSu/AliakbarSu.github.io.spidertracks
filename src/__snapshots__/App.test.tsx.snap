// Vitest Snapshot v1, https://vitest.dev/guide/snapshot.html

exports[`App Component > Should render correctly 1`] = `<App />`;
